### ISSUE TITLE
Accessibility: Consider keyboard shortcut disabling

### ISSUE THREAD

**Issue by rart** [craftersoftware/craftercms#8220]
### Latest version

- [x] The issue is in the develop branch
- [ ] The issue is in the latest released version
- [ ] The issue is in an older release (specified in the ticket)

### Duplicates

- [x] I have searched the existing issues

### Is your enhancement request related to a problem? Please describe.

[WCAG SC 2.1.4](https://w3.org/TR/WCAG21#character-key-shortcuts) suggests non-focus dependant shortcuts to allow disabling. Current shortcuts are not configurable or allow disabling. Some should be better implemented as focus-dependant (e.g. disable `e` shortcut when a dialog is opened.)

### Describe the solution you'd like

Consider enabling configuration based shortcuts (disabling) and access if any of the current should be focus-dependant.

**Comment by yacdaniel** [comment]
Verified fixed in `v5`

<img width="465" height="905" alt="Image" src="https://github.com/user-attachments/assets/5fab8ff9-ecfe-4396-9621-ef2c46286ab0" />

**Comment by yacdaniel** [comment]
Verified in `4.x`
